ReplayKitExample/BroadcastExtension/ExampleReplayKitAudioCapturer.h

```diff
@@ -6,7 +6,7 @@
 //
 
 #import <ReplayKit/ReplayKit.h>
-#import <TwilioVideo/TwilioVideo.h>
+#import <TwilioVideo/TVIAudioDevice.h>
```

Comment:
On Xcode 11 I was unable to compile unless I imported minimal headers. The intermittent "abort trap code 6" error occurred all the time.